src/app/core/components/item-nav/item-nav.component.ts

```diff
@@ -126,11 +126,13 @@ export class ItemNavComponent implements OnInit, OnDestroy {
   ngOnInit() {
 
     this.subscriptions.push(
+
+      // This first subscription only follow change in the current item id and use switch map to cancel previous requests
```

Comment:
This comment makes the one line 137 redundant, maybe remove the old one?

Reply:
done